portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.data;
+
+/* Class that represents how a comment object */
+
+public class Comment {
+  private final String username;
+  private String text;
+
+  public Comment(String username, String text) {
+    this.username = username;
+    this.text = text;
+  }
+
+  public String getUsername() {
+    return username;
+  }
+
+  public String getText() {
+    return text;
+  }
+
+  public String setText() {
```

Comment:
Unused. It's probably a good idea to only add methods and helpers as you need them.

This makes it clear to the reviewer why you're adding them (because you need them). And who knows, you might not even need it. This is called 'dead on arrival code', because it's dead (not used, except maybe in tests) from the moment it is introduced (and it's generally a code smell/antipattern).

Reply:
I was thinking about adding and edit feature to the comment section in the future, but I will add it then